corehq/apps/app_manager/dbaccessors.py

```diff
@@ -14,6 +14,8 @@
 from corehq.util.quickcache import quickcache
 from quickcache.django_quickcache import tiered_django_cache
 from corehq.toggles import VELLUM_SAVE_TO_CASE
+from corehq.apps.es.apps import app_id as app_id_filter
```

Comment:
Nit: How about
```python
from corehq.apps.es import apps as app_es
```
and then use it as
```python
app_es.app_id
```
Not sure if this is clearer or not. What do you think?